debian/matcher.go

```diff
@@ -0,0 +1,56 @@
+package debian
+
+import (
+	version "github.com/knqyf263/go-deb-version"
+	"github.com/quay/claircore"
+	"github.com/quay/claircore/libvuln/driver"
+)
+
+type Matcher struct{}
```

Comment:
You should pop a static verification of the interface in here, like `var _ driver.Matcher = (*Matcher)(nil)`, because I think this has the old names.

Reply:
:+1: